src/libluau.zig

```diff
@@ -974,18 +974,20 @@ pub const Lua = struct {
 
     /// Converts the Lua string at the given `index` to a string atom.
     /// The Lua value must be a string.
-    pub fn toStringAtom(lua: *Lua, index: i32, atom: *c_int) ![*:0]const u8 {
-        if (c.lua_tostringatom(lua.state, index, atom)) |ptr| {
-            return ptr;
+    pub fn toStringAtom(lua: *Lua, index: i32) !struct { i32, [:0]const u8 } {
+        var atom: c_int = -1;
```

Comment:
Is there a reason you use -1 here opposed to `undefined`? I see Luau uses -1 for unknown/invalid, so I guess it is okay to set it to -1 ahead of time. I think `undefined` shows a clear intent more. Otherwise it may seem like passing -1 into the function has meaning.